.travis.yml

```diff
@@ -47,7 +47,21 @@ matrix:
       script:
         # Streaming cpp test.
         - if [ $RAY_CI_STREAMING_CPP_AFFECTED == "1" ]; then ./ci/suppress_output bash streaming/src/test/run_streaming_queue_test.sh; fi
-        - if [ RAY_CI_STREAMING_PYTHON_AFFECTED == "1" ]; then python -m pytest -v --durations=5 --timeout=300 python/ray/streaming/tests/; fi
+        - if [ RAY_CI_STREAMING_PYTHON_AFFECTED == "1" ]; then python -m pytest -v --durations=5 --timeout=300 streaming/python/tests/; fi
+
+    - os: linux
+      env:
+        - TESTSUITE=streaming JDK='Oracle JDK 8' PYTHON=3.5 PYTHONWARNINGS=ignore RAY_INSTALL_JAVA=1
```

Comment:
Maybe we can put Java and Python tests in one CI job? So we don't need to compile C++ code twice.

Reply:
Good suggestion, fixed it